core/domain/entities/editor/Block.php

```diff
@@ -20,78 +17,60 @@
  * @author  Brent Christensen
  * @since   $VID:$
  */
-abstract class EditorBlock implements EditorBlockInterface
+abstract class Block implements BlockInterface
 {
 
-    const NS = 'event-espresso/';
-
     /**
-     * @var DomainInterface $domain
+     * BlockAssetManager that this editor block uses for asset registration
+     *
+     * @var BlockAssetManagerInterface $block_asset_manager
      */
-    protected $domain;
+    protected $block_asset_manager;
 
     /**
-     * @var LoaderInterface $loader
+     * @var array $attributes
      */
-    protected $loader;
-
+    private $attributes;
 
     /**
-     * @var Registry
+     * If set to true, then the block will render its content client side
+     * If false, then the block will render its content server side using the renderBlock() method
+     *
+     * @var bool $dynamic
      */
-    protected $assets_registry;
+    private $dynamic = false;
 
     /**
      * @var string $editor_block_type
      */
     private $editor_block_type;
 
-    /**
-     * @var WP_Block_Type $wp_block_type
-     */
-    private $wp_block_type;
-
     /**
      * @var array $supported_post_types
      */
     private $supported_post_types;
 
     /**
-     * @var array $attributes
-     */
-    private $attributes;
-
-    /**
-     * If set to true, then the block will render its content client side
-     * If false, then the block will render its content server side using the renderBlock() method
-     *
-     * @var bool $dynamic
+     * @var WP_Block_Type $wp_block_type
      */
-    private $dynamic = false;
+    private $wp_block_type;
 
 
     /**
-     * EditorBlockLoader constructor.
+     * BlockLoader constructor.
      *
-     * @param DomainInterface $domain
-     * @param LoaderInterface $loader
-     * @param Registry        $assets_registry
+     * @param BlockAssetManagerInterface $block_asset_manager
      */
-    public function __construct(
-        DomainInterface $domain,
-        LoaderInterface $loader,
-        Registry $assets_registry
-    ) {
-        $this->domain = $domain;
-        $this->loader = $loader;
-        $this->assets_registry = $assets_registry;
+    public function __construct(BlockAssetManagerInterface $block_asset_manager)
+    {
+        $this->block_asset_manager = $block_asset_manager;
     }
 
 
     /**
      * @return string
      */
-    public function editorBlockType()
+    public function blockType()
     {
         return $this->editor_block_type;
```

Comment:
should this property name change?

Reply:
ya good catch... I missed that. will change